### PR TITLE
2023-05-23 meeting notes

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
cc @nodejs/loaders

@isaacs @jlenon7 would you like me to add you to @nodejs/loaders so that you get team pings? That way I won’t have to try to remember to cc you explicitly.

@arcanis Note in here the discussion of renaming `registerLoader` to just `register`, as in `import { register } from 'node:module'`, to create precedent for a future `import { register } from 'node:fs'` and similar for other namespaces.

### CODE REVIEW

**Comment by jlenon7** [comment]
Sure @GeoffreyBooth

**Comment by isaacs** [comment]
@GeoffreyBooth sure!

**Comment by GeoffreyBooth** [comment]
Done. If someone would be so kind, this PR needs an approval, as does #144.

File: doc/meetings/2023-05-23.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+* Jacob Smith @JakobJingleheimer
+
+
+## Notes
+
+### https://github.com/nodejs/node/pull/46826 `registerLoader()`
+
+- Whether to deprecate/remove `--loader` flag. Something we might consider after landing `registerLoader`, to simplify overall API. (Loaders can just tell their users to swap `--loader` for `--import`).
+
+- Name align on `register`, so `import { register } from 'node:module'` for what’s currently called `registerLoader`, to provide a standard namespace for registering customization hooks for various APIs. For example, in the future:
```
Comment (JakobJingleheimer) [inline doc/meetings/2023-05-23.md:23]:
```suggestion
- Align name on `register`, so `import { register } from 'node:module'` for what’s currently called `registerLoader`, to provide a standard namespace for registering customization hooks for various APIs. For example, in the future:
```

**Review by JakobJingleheimer — APPROVED** [review]

**Review by JakobJingleheimer — APPROVED** [review]

**Pull request merged**